aws/resource_aws_sns_platform_application.go

```diff
@@ -203,6 +203,11 @@ func resourceAwsSnsPlatformApplicationRead(d *schema.ResourceData, meta interfac
 	})
 
 	if err != nil {
+		if isAWSErr(err, sns.ErrCodeNotFoundException, "") {
+			log.Printf("[WARN] SNS Topic (%s) not found, error code (404)", d.Id())
```

Comment:
Minor copy-paste fix 😄 

```suggestion
			log.Printf("[WARN] SNS Platform Application (%s) not found, removing from state", d.Id())
```

Reply:
woops! Fixed